srcpkgs/cloudflared/update

```diff
@@ -0,0 +1 @@
+site=https://github.com/cloudflare/cloudflared/releases
```

Comment:
Is this really needed? update-check detects 2022.10.3 for me without this.

Reply:
Definitely not if it works already :D I just thought with an update file, the package would get automatically updated?

Reply:
The update file is used for [checking for new upstream releases](https://github.com/void-linux/void-packages/blob/master/Manual.md#checking-for-new-upstream-releases), every day all packages are checked and the result is available on https://repo-default.voidlinux.org/void-updates/void-updates.txt. Packages do not ever get automatically updated, they have to be bumped and tested manually. Updating them automatically would only lead to low quality packages and a lot of issues.

Reply:
All right, thanks for the detailed explanation :)
Just removed the file.